### DISCUSSION TITLE
while using sudo dpkg-buildpackage -us -uc command i am facing mentioned issue

### DISCUSSION THREAD

**Discussion by balajin2022** [rackslab/Slurm-web#244] [Q&A]
File "/home/balajin/slurm-web/.pybuild/cpython3_3.10/build/slurmweb/restapi/slurmrestapi.py", line 23, in <module>
    import pyslurm
ModuleNotFoundError: No module named 'pyslurm'
![Screenshot from 2024-01-19 14-29-04](https://github.com/rackslab/slurm-web/assets/103120252/4ed5815e-1343-48c6-9de2-28629050b5bc)

**Comment by rezib** [comment]
Hello @balajin2022,

I think I should state it more clearly in contributing guidelines or with issue templates but issues should be reserved in this project to **actual bug reports** and **feature requests**.

You have opened #239, #241 and #243 in just a couple of days, none of them enter in these categories. These are  support requests. Community support requests have a much in better place in Q&A section of project discussions: https://github.com/rackslab/slurm-web/discussions/categories/q-a

Can you please stop creating issues for questions that are not actual bugs?

Please note that if nobody answers your questions, maybe nobody has the answer. Spamming with new issues or new discussions will not help you to get helpful answers.

I will convert this issue into a Q&A discussion and close #241 with this comment. Please use this new unique discussion to report your progress and potential new questions.

**Comment by rezib** [comment]
This question was about deprecated Slurm-web 2, then I close the discussion.
